src/main/java/service/TicketService.java

```diff
@@ -0,0 +1,81 @@
+package service;
+
+import DAO.TicketDAO;
+import model.*;
+
+import model.enums.TicketType;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Component;
+import org.springframework.transaction.annotation.Transactional;
+import view.Printable;
+
+import java.util.List;
+import java.util.UUID;
+
+
+@Component
+@Transactional
+public class TicketService implements Printable {
+
+    @Autowired
+    private TicketDAO ticketDAO;
+
+    @Value("${isUsersUpdateAndTicketsCreationAvailable}")
+    private boolean isTicketsCreationAvailable;
```

Comment:
you use the same variable from `application.properties`, in `TicketService` and in `UserService`.
It is better to use separate switcher on/off for each case, or unite this function " update User and create ticket" in 1 single method. For example:
```java
 @Transactional
    public void updateUserAndCreateTicket(User user, Ticket ticket) {
        if (!isUsersUpdateAndTicketsCreationAvailable) {
            throw new ... ;
        }
        userDAO.updateUserStatusById(user);
        ticketDAO.save(ticket);
    }
```

Also this example covers the p.3 from HW (I didn't find its' implementation into the code, ping me if I'm wrong).
Also you can perform p.3 from HW task with 2nd approach:
1. on service layer create method:
```java
    public void updateUserAndCreateTicket(User user, Ticket ticket) {
        if (!isUsersUpdateAndTicketsCreationAvailable) {
            throw new ... ;
        }
        userDAO.updateUserAndCreateTicket(user, ticket);
    }
```
2. on dao layer create transactional method with 2 queries:
```java
@Transactional
public void updateUserAndCreateTicket(User user, Ticket ticket){
      jdbcTemplate.update(UPDATE_USER_STATUS, variables);
      jdbcTemplate.update(CREATE_TICKET, variables);
}
```

Reply:
Fixed. I united this function in new service 'ApplicationService'